ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -997,6 +997,15 @@ extension BrowserViewController: WKNavigationDelegate {
 
     rewards.reportTabNavigation(tabId: tab.rewardsId)
 
+    // Notify of tab changes after navigation completes but before notifying that
+    // the tab has loaded, so that any listeners can process the tab changes
+    // before the tab is considered loaded.
+    rewards.maybeNotifyTabDidChange(
```

Comment:
Moved `NotifyTabDidChange` and `NotifyTabDidLoad` notifications to `webView(_:didCommit:)` to align with the Desktop implementation.